src/lisp/kernel/init.lsp

```diff
@@ -855,6 +856,7 @@ the stage, the +application-name+ and the +bitcode-name+"
                  `(my-do-time #'(lambda () ,form))))
            t)
 (export 'my-time)
+)
```

Comment:
yes, it can be dropped, but @drmeister may want to copy it somewhere else. please confirm that you noted this, and then i'll amend the PR to delete it.